service-runner/src/main/kotlin/org/stellar/anchor/platform/TestProfileRunner.kt

```diff
@@ -76,24 +74,60 @@ class TestProfileExecutor(val config: TestConfig) {
       // Start servers
       val jobs = mutableListOf<Job>()
       val scope = CoroutineScope(Dispatchers.Default)
-      jobs += scope.launch { ServiceRunner.startKotlinReferenceServer(envMap, false) }
-      jobs += scope.launch { runningServers.add(ServiceRunner.startAnchorReferenceServer()) }
-      jobs += scope.launch { runningServers.add(ServiceRunner.startStellarObserver(envMap)) }
-      jobs += scope.launch { runningServers.add(ServiceRunner.startSepServer(envMap)) }
-      jobs.forEach { it.join() }
-
-      if (wait) {
-        while (true) {
-          delay(60000)
+
+      if (shouldStartAllServers || shouldStartKotlinReferenceServer) {
+        info("Starting Kotlin reference server...")
+        jobs += scope.launch { ServiceRunner.startKotlinReferenceServer(envMap, false) }
+      }
+      if (shouldStartAllServers || shouldStartReferenceServer) {
+        info("Starting Java reference server...")
+        jobs += scope.launch { runningServers.add(ServiceRunner.startAnchorReferenceServer()) }
+      }
```

Comment:
If `shouldStartAllServers = true`, this code will start both Kotlin and Java Reference Server. Won't this be a problem?

Reply:
No it is not. We can start both reference servers.